ext_conf_template.txt

```diff
@@ -10,6 +10,9 @@ jobGroupPattern = bms_jobno_%s
 # cat=basic/enable; type=string; label=Name of the group to use as a template (you need to create a group with this name!)
 jobGroupTemplate = bms department template %s
 
+# cat=basic/enable; type=boolean; label=Do NOT use backend user credentials for emails (name and mail address)
+disableBackendUserCredentialsForEmails =
```

Comment:
Can we rename this to `useBackendUserCredentialsForEmails = 1`?

TYPO3 already has too many negated fields which cause cognitive overhead. ;-)

Reply:
The problem is that I want to keep the functionality backwards-compatible.

If people update without changing the extension configuration this option will be missing (and therefore resolve to false), disabling BE user credentials for those installations.
The negated option does not have those issues as false is the default.